Replace five SPKs (10+Mb) with one SPK (0.5Mb)

drbitboy opened the pull request:
This gets us down below 8Mb of kernel bandwidth for testing; the changes to SpiceyPy are done, in the pipeline, and waiting for this one.

drbitboy committed:
code to create minimal SPK for SpiceyPy unit tests

drbitboy committed:
converted allspk.bsp to allspk_{big,little}endian.bsp

drbitboy committed:
fix markdown in code/README.md

drbitboy committed:
Fix more items in code/README.md

drbitboy committed:
Fix yet more items in code/README.md

AndrewAnnex commented:
I am going to hold off considering this until after the next release. The reason being that the kernel data is small enough as is, and there are more important things to fix/improve/add.

drbitboy commented:
Good plan.  I'll do the same thing to the CKs (currently 6Mb+) if I have a chance and commit that to this branch/pr, so it can all go in at once.

Note that in setting this up I found a typo/bug in test_wrapper.py where it uses spice.spksub (lines 6081, 6089, 6213 in your current master d41698d; compare to my branch improvement/one-small-spk  a695bba).  I'll put a pull request together with just that for the next release.

drbitboy commented:
FYI:  this does not change any of the existing kernels; it only adds 10 files that will be ignored by, and in no way affect, the current SpiceyPy tree or its testing.

So merging this PR risks nothing, and makes it easier for me to continue working in parallel.

Just sayin'

;-)

drbitboy committed:
Add small CK CKSLICERed from Cassini CK; tweak small SPK scripts

allck.bash
allck.bc

- Script allck.bash creates CK allck.bc
- Extract enough data from 13056_13057ra.bc to support SpiceyPy unit tests

code/Makefile

- Make this more general

code/spkpvn.c.n0066.patched
code/spkpvn.c.patch

- Make inserted section more compact
- Remove blank line

drbitboy committed:
Merge pull request #2 from drbitboy/improvement/one-small-ck

Add small CK CKSLICERed from Cassini CK; tweak small SPK scripts

drbitboy committed:
Added script to convert 3.9Mb Binary PCK to 8kb for SpiceyPy testing

drbitboy committed:
Merge pull request #3 from drbitboy/improvement/one-small-bpck

Added script to convert 3.9Mb Binary PCK to 8kb for SpiceyPy testing
